viite-UI/manual/index.md

```diff
@@ -456,99 +529,121 @@ _Kuva 7.8: Manuaalinen kääntö siirron tai numeroinnin yhteydessä_
+Ensisijaisesti näytetään tiedot projektissa vielä käsittelemättömistä tieosoiteprojektille varatuista linkeistä. Nämä näytetään oranssilla värillä.
+Kun kaikki projektiin varatut linkit on käsitelty jollain toimenpiteellä, näytetään muut Viiteen havaitsemat ongelmat tieverkolla. 
 
 ![Tarkastusilmoitusnäkymä](k49.png)
 
-_Kuva 7.11: Tarkastusilmoitukset näkyvät projektissa oikealla._
+_Kuva 7.9: Tarkastusilmoitukset näkyvät projektissa oikealla._
 
 Tarkastusilmoitus koostuu seuraavista kentistä:
 
 |Kenttä|Kuvaus|
 |------|------|
 |Linkids|Tarkastuksen kohteena oleva yksittäisen tielinkin ID. Vaihtoehtoisesti linkkien lukumäärä, jos tarkastusilmoitus koskee useampaa linkkiä.|
-|Virhe|Kuvaus tarkastuksen havaitsemasta ongelmatilanteesta.|
-|Info|Mahdollisia lisäohjeita tarkastuksen virhetilanteen korjaamiseksi.|
+|Virhe  |Kuvaus tarkastuksen havaitsemasta ongelmatilanteesta.|
+|Info   |Mahdollisia lisäohjeita tarkastuksen virhetilanteen korjaamiseksi.|
 
 ![Tarkastusilmoitus](k50.JPG)
 
-_Kuva 7.12: Tielinkki 7304402 on tieosan viimeinen linkki, mutta siltä puuttuu jatkuvuuskoodi Tien loppu._ 
+_Kuva 7.10: Tielinkki 7304402 näyttää olevan tieosan viimeinen linkki, mutta sen jatkuvuuskoodi on joku muu kuin _Tien loppu_._ 
 
-Karttanäkymä kohdistuu tarkastusilmoituksen kohteena olevan tielinkin keskikohtaan painamalla Korjaa-painiketta. Klikkaamalla painiketta uudestaan kohdistus siirtyy seuraavaan tielinkkiin, jos sama tarkastus kohdistuu useampaan linkkiin. Käyttäjä voi valita tarkastusilmoituksen kohteena olevan tielinkin ja tehdä sille korjaavan toimenpiteen. 
+Painamalla _Korjaa_-painiketta karttanäkymä kohdistuu tarkastusilmoituksen kohteena olevan tielinkin keskikohtaan. Klikkaamalla painiketta uudestaan kohdistus siirtyy seuraavaan tielinkkiin, jos sama tarkastus kohdistuu useampaan linkkiin. Käyttäjä voi valita tarkastusilmoituksen kohteena olevan tielinkin ja tehdä sille korjaavan toimenpiteen. 
 
 #### Tieosoiteprojektissa tehtävät tarkastukset:
 
 Tieosoiteprojektiin kohdistuvat tarkastukset:
 
+- Kaikille projektiin varatuille tielinkeille tulee tehdä jokin toimenpide
 - Uusi tieosa ei saa olla varattuna jossakin toisessa tieosoiteprojektissa
-- Kaikki projektiin varatuille tielinkeille tulee tehdä jokin toimenpide
 - Tieosoitteen kasvusuunta ei saa muuttua kesken tien
-- Tieosoitteellinen tie ei saa haarautua muuten kuin ajoratakoodivaihdoksessa
+- Tieosoitteellinen tie ei saa haarautua muuten kuin ajoratakoodin vaihtuessa
 - Ajoratojen 1 ja 2 tulee kattaa samaa osoitealue
-- Tieosoitteelta ei saa puuttua tieosoiteväliä (katkoa m-arvoissa)
+- Tieosoitteelta ei saa puuttua tieosoiteväliä (katkoa m-arvoissa) <!-- <- !!!TARKISTETTAVA!!! -->
```

Comment:
eikö tää ole ihan totta 😮

Reply:
Pointti, ei _saa_. Tällä hetkellä (tai ihan äskettäin inkin) pystyy tekemään sellaisen osoitteistuksen, että katko osoitteistukseen tulee. Mutta ei se tarkoita, että niin saa tehdä. Ei saa. :)